keep-only-warm-data/docker/kuzzle/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM kuzzleio/kuzzle:develop
 
-ADD custom_start.sh /custom_start.sh
+MAINTAINER amaret@kuzzle.io
```

Comment:
Ok, I am really a nitpicker but [`MAINTAINER` is deprecated in favor of LABEL](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).

Reply:
No problem, that's how you learn the most ;)